app-typescript/components/Avatar.tsx

```diff
@@ -12,7 +13,15 @@ interface IImageAvatar extends IBase {
 interface ITextAvatar extends IBase {
     text: string; // limited to 3 characters
 }
-
+interface INumberAvatar extends IBase {
+    number: string;
+}
+interface IIconAvatar extends IBase {
+    status?: 'default' | 'in-progress' | 'done';  // defaults to default
```

Comment:
Same with `status`, can be simply use the same `type` from the `Icon` component (and let the Planning module choose which colour to apply?)